src/net/gateway-pool.ts

```diff
@@ -92,6 +94,22 @@ export class GatewayPool {
         return this.activeGateway.isReady
     }
 
+    public getArchiveUri(): string {
+        return this.activeGateway.getArchiveUri()
+    }
+
+    public setArchiveUri(uri: string) {
+        this.activeGateway.setArchiveUri(uri)
+    }
+
+    public getEnvironment(): VocdoniEnvironment {
```

Comment:
`getEnvironment` and `getNetwork` are redundant. We have getters for this.

To do the resolution in the event that we don't have this value, then we might call this `resolveArchiveUri`? 🤔 

Currently the getters return the value immediately, but this would need to return a Promise